docs/wallets/wallets.mdx

```diff
@@ -63,27 +64,26 @@ private key phrases stored in a file; something like `keyfile.bin` or as a long
 and numbers.
 
 Private keys allow for full access to an account's contents and should be protected. These keys are
-never needed to diagnose a Hotspot or on-chain transaction.
+never needed to diagnose a Hotspot or verify on-chain transaction.
```

Comment:
addition of "verify" should make "transaction" plural?
"verify on-chain transactions"